server/routes/public.js

```diff
@@ -29,15 +29,48 @@ router.get('/api/login', async (req, res) => {
 });
 
 router.get('/api/create-user', async (req, res) => {
-    const firstName = req.query.firstName;
-    const lastName = req.query.lastName;
-    const type = req.query.type; //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
-    const studentID = req.query.studentID;
-    const email = req.query.email;
-    const password = req.query.password;
-    const phone = req.query.phone;
-
-    const user = await accountHandler.createUserAccount(firstName, lastName, type, studentID, email, password, phone);
+    // joi verification
+    // Validate appropriate parameters are passed into the create account endpoint.
+    const paramSchema = Joi.object({
+       
+       //The following joi verifications make sure the values inputted adhere to the format requirements
+       //specified in the migrations> create-worker-table and migrations> create-student-table.
+
+        email: Joi.string().email({ minDomainSegments: 2, tlds: { allow: ['com', 'net', 'ca'] } }).max(320).required(),
+        password: Joi.string().min(3).max(40).required(),
+        phone: Joi.string().max(20).phoneNumber(),
+        first_name: Joi.string().min(1).max(20).required(),
+        last_name: Joi.string().min(1).max(20).required(),
```

Comment:
Hi Nancy,
As mentioned above, I would recommend adjusting these names to lowerCamelCase to maintain code consistency.

Reply:
Hi Amy,
Thank you for pointing that for me!
My action: I've changed the JOI verification names to be in camelCase.
You can now see these changes in lines  40-48 of `server/routes/public.js`

Please confirm if this takes care of your suggestion!

Reply:
Hi Nancy, 
Yes, that works. Thank you for making those changes!